feature/gnoi/os/tests/osinstall/osinstall_test.go

```diff
@@ -254,6 +257,33 @@ func (tc *testCase) transferOS(ctx context.Context, t *testing.T, standby bool)
 	}
 }
 
+func (tc *testCase) isSupervisorReady(ctx context.Context, t *testing.T) error {
+	var counter int8
+	const maxRetries = 15
+	for i := int8(1); i < maxRetries; i++ {
+		if counter == maxRetries {
+			return fmt.Errorf("OS.Verify RPC did not report the standby supervisor state as available")
+		}
+		counter++
+		r, err := tc.osc.Verify(ctx, &ospb.VerifyRequest{})
+		if err != nil {
+			t.Fatal(err)
+		}
+		switch v := r.GetVerifyStandby().GetState().(type) {
+		case *ospb.VerifyStandby_StandbyState:
```

Comment:
you only have one case here, do you still need the switch?

Reply:
I thought it increased readability. What suggestions might you have to replace the switch? If require, I could change to if/else, but that might be a little more complex to read.